File: video/v1/published_track/fetch-default/fetch-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Video.V1.Room.Participant;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var publishedTrack = PublishedTrackResource.Fetch(
            pathRoomSid: "RMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathParticipantSid: "PAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathSid: "MTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(publishedTrack.Name);
    }
}

File: video/v1/published_track/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

published_track = @client.video
  .rooms('RMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .participants('PAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .published_tracks('MTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .fetch

puts published_track.name
